src/nb_enum.cpp

```diff
@@ -71,7 +71,7 @@ PyObject *enum_create(enum_init_data *ed) noexcept {
     scope.attr(name) = result;
     result.attr("__doc__") = ed->docstr ? str(ed->docstr) : none();
 
-    result.attr("__str__") = enum_mod.attr("Enum").attr("__str__");
```

Comment:
Can you use `factor_name` (and otherwise `Enum`) only when this is a flag?

Reply:
Done.